modules/twinklespeedy.js

```diff
@@ -790,17 +790,17 @@ Twinkle.speedy.userNonRedirectList = [
 		value: 'notwebhost',
 		tooltip: 'Pages in userspace consisting of writings, information, discussions, and/or activities not closely related to Wikipedia\'s goals, where the owner has made few or no edits outside of userspace, with the exception of plausible drafts, pages adhering to WP:UPYES, and résumé-style pages.'
 	},
-	{
-		label: 'G6: Blank draft',
-		value: 'blankdraft',
-		tooltip: 'Userspace drafts containing only the default Article Wizard text, where the author of the page has been inactive for at least one year.',
-		hideWhenMultiple: true
-	},
 	{
 		label: 'G11: Promotional user page under a promotional user name',
 		value: 'spamuser',
 		tooltip: 'A promotional user page, with a username that promotes or implies affiliation with the thing being promoted. Note that simply having a page on a company or product in one\'s userspace does not qualify it for deletion. If a user page is spammy but the username is not, then consider tagging with regular G11 instead.',
 		hideWhenMultiple: true
+	},
+	{
+		label: 'G13: Old, abandoned AfC submission or blank draft',
+		value: 'afc',
+		tooltip: 'Any rejected or unsubmitted AfC submission, or a blank draft, unedited in over 6 months,',
```

Comment:
This implies only blank drafts are eligible which is not true.

Reply:
Like above, I don't see how the wording implies that. What wording are you suggesting?